.jenkins/env.sh

```diff
@@ -9,6 +9,7 @@ if [ -d "${envdir}" ] ; then
     popd
 else
     `git clone https://github.com/VulcanClimateModeling/buildenv ${envdir} &>/dev/null`
+    `cd ${envdir} && git checkout slurmvar &>/dev/null`
```

Comment:
What are the `s here for?

Reply:
Good question, that style was already in this file from another repo. It just indicates run this command i think, though may be unnecessary.